lib/modelHelpers.py

```diff
@@ -93,12 +109,12 @@ def format_clipboard():
         return format_message(clip.text())
 
 
-def generate_payload(
+def send_request(
```

Comment:
We still have a `gpt_send_request` function so I think the similar names of both `send_request` and `gpt_send_request` is a bit confusing. If we are just calling `gpt_send_request` from `send_request` and the former is a small function, we can probably just move it inside.

Reply:
That being said I might be missing something since I didn't look through the entirety of the codebase in relation to this change.

Reply:
moved